neo/sys/posix/posix_main.cpp

```diff
@@ -688,11 +688,7 @@ Posix_Cwd
 const char* Posix_Cwd()
 {
 	static char cwd[MAX_OSPATH];
```

Comment:
It really doesn't make sense to make it dynamically, neither here nor in Sys_EXEPath().
Those paths are used as a base to open game assets/configs/etc - by copying it into a buffer of `MAX_OSPATH` size and then appending `/base/default.cfg` or whatever.
So as the game does not use strings of dynamic sizes elsewhere to handle paths, doing it here doesn't really improve anything.